### PR TITLE
[WFLY-14762] Expose complete-timeout Infinispan transaction setting

### PR DESCRIPTION
**Pull request by spyrkob**
Issue: https://issues.redhat.com/browse/WFLY-14762

### CODE REVIEW

**Comment by spyrkob** [comment]
@pferraro could you review when you have a moment?

**Review by pferraro — CHANGES_REQUESTED** [review]

**Comment by spyrkob** [comment]
Thanks @pferraro, changes made.

**Comment by spyrkob** [comment]
@pferraro does this change need a RFE doc as well?

**Comment by pferraro** [comment]
> @pferraro does this change need a RFE doc as well?

No - this is a bug fix, not an enhancement.

File: clustering/infinispan/extension/src/main/resources/schema/jboss-as-infinispan_13_0.xsd [hunk 1]
```diff
@@ -330,6 +330,14 @@
                 <xs:documentation>The locking mode for this cache, one of OPTIMISTIC or PESSIMISTIC.</xs:documentation>
             </xs:annotation>
         </xs:attribute>
+        <xs:attribute name="complete-timeout" type="xs:long" default="60000">
+            <xs:annotation>
+                <xs:documentation>
+                    The default duration (in ms) after which idle transactions are rolled back.
+                    Only applies to a BATCH transaction mode.
```
Comment (pferraro) [inline clustering/infinispan/extension/src/main/resources/schema/jboss-as-infinispan_13_0.xsd:337]: This attribute applies to any transactional cache, not just BATCH mode.

**Review by pferraro — CHANGES_REQUESTED** [review]

**Review by pferraro — APPROVED** [review]

**Pull request merged**